all-in-one/Dockerfile

```
@@ -66,19 +66,13 @@ RUN if [ -n "$REST_PATH" ]; \
     mv oscal-rest-service-app/target/oscal-rest-service-app*.jar "/app/oscal-rest-service-app.jar"; \
     fi
 
-FROM openjdk:${JAVA_VERSION}-slim-bullseye
+FROM gcr.io/distroless/java11-debian11
```

```suggestion
FROM gcr.io/distroless/java${JAVA_VERSION}-debian11
```

I don't care too much about validating the `JAVA_VERSION` (our current image would fail if you provided some not-available-on-bullseye version). But I do like the ability to build with Java 17 (and I usually do when building for myself).